content/02.intro.md

```diff
@@ -23,7 +23,8 @@ algorithms have produced comparable or higher accuracy than previous
 best-in-class methods that required years of extensive customization, and
 specialized implementations are now being used at industrial scales.
 
-Deep learning approaches grew from research in neural networks, which were first proposed in 1943 [@doi:10.1007/BF02478259] as a model
+Deep learning originated from the field of neural networks.
```

Comment:
This partially reverts a change that was made in #566. We'll need to do a full review before merging.  I may have time in the next 24 hrs if @cgreene doesn't do it first.

Reply:
Currently planning to do this before 10 AM tomorrow 🤞

Reply:
Removed this change.